architecture/consumption-architecture-dependency-management.md

```diff
@@ -0,0 +1,148 @@
+  * [vexctl](https://github.com/openvex/vexctl) is a flagship project under [OpenVEX](https://github.com/openvex). vexctl generates, merges and attest VEX statements.
+  * [OpenVEX](https://github.com/openvex) is a [sandbox](https://github.com/ossf/tac/blob/main/process/project-lifecycle-documents/openvex_for_sandbox_stage.md) project in the [Vulnerability Disclosures Working Group](https://github.com/ossf/wg-vulnerability-disclosures). It is a specification, a Go library and a set of tools for VEX. 
+* Digital Signing , Signature Verification, Transparency Log
+  * [Sigstore](https://github.com/sigstore) is an an OpenSSF [graduated](https://github.com/ossf/tac/blob/main/process/project-lifecycle-documents/sigstore_graduated_stage.md) project under [OpenSSF TAC](https://github.com/ossf/tac) directly. Sigstore public instance provides digital signing capability, OIDC PKI infrastructure, and a transparency  log infrastructure for OSS projects. Sigstore is the trust foundation for OSS digital signing using a short lived X.509 certificate. Sigstore transparency log provides an immutable, tamper-resistant ledger of artifact attestations. gitsign is a tool within Sigstore ecosystem that provides Git signing with OSS producers' own GitHub / OIDC identity.   
+
+### Ecosystem Support
+
+![Ecosystem Support](./images/OpenSSF_Practitioner_Framework_Ecosystem_Support.jpg)
+
+The digram illustrates the supports that OSS projects expect to get from the ecosystems that host the projects, and the typical patterns that OSS consumers in an organization adopt to manage software securities including dependency manageent. 
+
+### Frameworks, Specifications, Standards, Education and Training
+
+![State](./images/OpenSSF_Practitioner_Framework_state.jpg)
+![Frameworks, Specifications, Standards, Education and Training](./images/OpenSSF_Practitioner_Framework_Framework_Specifications_Standards_Education.jpg)
+
+OpenSSF and peer foundations hosts software security frameworks, specifications, standards that propels the development of security tooling. This section focuses on the technologies that we have not got a chance to touch on in the previous sections:
+* [S2C2F](https://github.com/ossf/s2c2f) is an an OpenSSF [incubating](https://github.com/ossf/tac/blob/main/process/project-lifecycle-documents/s2c2f_incubation_stage.md) project in the [Supply Chain Integrity Working Group](https://github.com/ossf/wg-supply-chain-integrity). It's for OSS consumers' organizations. The framework defines how to securely consume OSS dependencies by the developers.
+* [SBOMit](https://github.com/sbomit) is a [sandbox](https://github.com/ossf/tac/blob/main/process/project-lifecycle-documents/SBOMit_sandbox_stage.md) project in OpenSSF [Security Tooling Working Group](https://github.com/ossf/wg-security-tooling). It standardizes the naming and location of SBOMs.
```

Comment:
```suggestion
* [SBOMit](https://github.com/sbomit) is a [sandbox](https://github.com/ossf/tac/blob/main/process/project-lifecycle-documents/SBOMit_sandbox_stage.md) project in OpenSSF [Security Tooling Working Group](https://github.com/ossf/wg-security-tooling). It standardizes an SBOM format agnostic method for attesting components with additional verification information.
```

Reply:
Thank you @jkjell for making updates! really appreciate it!